test/functional/apps/discover/ccs_compatibility/_data_view_editor.ts

```diff
@@ -63,7 +52,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('allows creating a new data view', async function () {
       const dataViewToCreate = config.get('esTestCluster.ccs') ? 'ftr-remote:logstash' : 'logstash';
-      await createDataView(dataViewToCreate);
+      await PageObjects.discover.createAdHocDataView(dataViewToCreate, true);
```

Comment:
Hi @dmlemeshko,

I would keep the previous logic here as we probably want to test persisting this new data view.

Reply:
Hey @jughosta 

I reverted back the function but since it is copied in other 4 test files, I moved it to discover_page. Hope you are ok with it.

I didn't realise `createAdHocDataView` has few differences (2 retry checks) vs `createDataView`. But I still believe it would be great for other teams to have a more unified logic so folks don't create its own "createDataView" function for different test cases.